test/runtests.jl

```diff
@@ -85,51 +85,83 @@ function test_qst_hml(ρ, obs; n=10_000, β=0.0, asymptotic=false, alt=false, ma
 end
 
 
-function test_qpt_ml(n=1000;ρ=zeros(Float64,0,0))
+function test_qpt_free_lsq(n=1000; E=zeros(Complex128,0,0))
 
-    obs  = Matrix[ [0 1; 1 0], [0 -1im; 1im 0], pauli(3) ]
-    prep = map(projector,Vector[ [1;0], [0;1], 1/sqrt(2)*[1;1], 1/sqrt(2)*[1;-1], 1/sqrt(2)*[1;-1im], 1/sqrt(2)*[1;1im] ] )
-    
-    exps = Matrix[ choi_liou_involution(vec(o)*vec(p)') for o in obs, p in prep ]
+    prep = map(projector, Vector[ [1,0],
+                                  [0,1],
+                                  1/sqrt(2)*[1,1],
+                                  1/sqrt(2)*[1,-1],
+                                  1/sqrt(2)*[1,-1im],
+                                  1/sqrt(2)*[1,1im] ] )
 
-    pred = reduce(vcat, map(m->vec(m)', vec(exps)) )
-    
-    if trace(ρ) == 0.0
-        ρ = choi_liou_involution(liou(rand_unitary(2))) 
-    end
-    
-    asymptotic_means = real(pred*vec(ρ))
-
-    samples = Float64[ rand(Bernoulli((μ+1)/2),n) for μ in asymptotic_means ]
-    sample_mean = 2*map(mean,samples)-1
-    sample_var  = 4*map(var,samples)/n
-    
-    ρest, obj, status = qpt_ml(pred, sample_mean, sample_var);
-    
-    # get the normalization right
-    choi_err = ρ-ρest |> choi_liou_involution |> liou2choi
-
-    println("Status                  : $(status)")
-    println("Diamond norm lower bound: $(snorm(choi_err,1))")
-    println("χ² error                : $(obj)")
-    println("Eigvals ρ:")
-    for ev in eigvals(ρ)
-        println(ev)
+    if size(E) == (0,0)
+        E = liou(rand(RandomQuantum.ClosedHaarEnsemble(2)))
     end
-    println("Eigvals ρest:")
-    for ev in eigvals(ρest)
-        println(real(ev))
+
+    tomo = FreeLSProcessTomo(prep, prep)
```

Comment:
Do we want another test where the `states` and `obs` inputs are not the same, as in the prior version of this test?